src/blocks/input.rs

```diff
@@ -18,16 +18,16 @@ use crate::val_helpr::ValidationResult;
 /// [slack's guide to using modals 🔗]: https://api.slack.com/surfaces/modals/using#gathering_input
 #[derive(Clone, Debug, Deserialize, Hash, PartialEq, Serialize, Validate)]
 pub struct Contents {
-    #[validate(custom = "validation::text_max_len_2k")]
-    label: compose::Text,
+    #[validate(custom = "validate::label")]
+    label: text::Plain,
```

Comment:
the field types need to be text::Text for JSON serialization; text::Plain has no `type` property